bundles/override.rst

```diff
@@ -5,8 +5,8 @@ How to Override any Part of a Bundle
 ====================================
 
 This document is a quick reference for how to override different parts of
-third-party bundles without using :doc:`/bundles/inheritance`, which is
-deprecated since Symfony 3.4.
+third-party bundles without using :doc:`/bundles/inheritance`, which was
+removed in Symfony 4.0.
```

Comment:
Let's make this description generic to not require any more updates in later versions (and thus merge conflicts), e.g:

 > When using another bundle, you might want to customize or override some of its features. This document describes ways of overriding the most common features of a bundle.